app/lib/task/global_lock.dart

```diff
@@ -63,21 +63,19 @@ class GlobalLock {
           // Try to refresh, if claim is still valid and we're not done.
           if (c.valid && !done.isCompleted) {
             refreshed = c.refresh();
-            try {
-              if (!await refreshed) {
-                _log.warning(
-                  'failed to refresh claim $claimId on $_lockId',
-                );
-                return; // Stop refreshing, if refresh failed.
-              }
-            } on Exception catch (e, st) {
-              // Stop refreshing, and log the error
-              _log.severe('Error refreshing claim $claimId on $_lockId', e, st);
-              return;
+            if (!await refreshed) {
+              _log.warning('failed to refresh claim $claimId on $_lockId');
+              return; // Stop refreshing, if refresh failed.
             }
           }
         }
-      });
+      } on Exception catch (e, st) {
```

Comment:
Something like this should really only happen in `await refreshed`.

But I guess I don't really mind this.

You probably should consider also eating the exception in `await refreshed; // wait to any ongoing refreshing attempt is done` later...